.github/actions/python-ci/action.yml

```diff
@@ -108,6 +108,14 @@ runs:
       run: |
         python -m pip install --upgrade pip
         python -m pip install pytest==8.0.0
+        python -m pip install pyopengltk==0.0.4
+
+    - name: Install Tkinter (Linux)
+      if: runner.os == 'Linux'
+      shell: bash
+      run: |
+        apt update
+        apt install -y python3-tk
```

Comment:
alright, lets not worry about CI for now, its out of scope and require modifying the docker image as we should not apt install in CI logic ideally.